package.json

```diff
@@ -24,7 +24,7 @@
     "deep-equal": "^1.0.1",
     "dlnacasts": "^0.1.0",
     "drag-drop": "^2.12.1",
-    "electron": "1.4.0",
+    "electron": "^1.3.5",
```

Comment:
We have always specified the Electron version exactly, never with a range. Can you fix this?